docs/Coding-Standards/Lite-Page-JS.mdx

```diff
@@ -0,0 +1,17 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Coding Standards/Lite Page JS" />
+
+# Lite Page JS
+Our Lite pages are different to our canonical pages in that we do not have the React library available on the client; the library is only available to build the serverside response and React is stripped out of the client JS. With this in mind, we need a way to support limited dynamic functionality on lite pages; we should minimize this for two primary reasons:
+- Integrating JS is more complex and we do not benefit from bundle splitting offered by webpack in Express and NextJS' in built support
+- We want to minimize page weight, JS adds to this
+
+## Recommeded Approach
+It is important any inline JS is transpiled to ensure it works on our supported browsers as is the case for any JS we add to our canonical pages. To do so inline JS should be maintained in its own file as shown on [this PR](https://github.com/bbc/simorgh/pull/12360/files#diff-bc423afafae9fdc5e0704ecc79da153814074be3a9c5c88de299dde09667a5bc); this allows it be included in a way where transpilation can take place.
+
+The file can then be included an the serverside response via the `toString()` api as is shown on [this PR](https://github.com/bbc/simorgh/pull/12360/files#diff-7740db73fb905cc3cb804a52fc779531d18dab5bd04724335dd5237481149defR30); the inline script can be transpiled at build time ensuring the transpiled JS is included in the page response appropriately.
+
+By applying this approach we can use modern JS and Typescript when maintaining the code rather than inlining transpiled code directly in our React components and maintianing it as a string.
+
+Additionally we are able to reliably test this code as we do any other code we maintain as shown in [this PR](https://github.com/bbc/simorgh/pull/12360/files#diff-a2444976147693573560a81ce6e248fa83e719d9474f021ba94707f0225560c4R2).
```

Comment:
As above, this code has now moved on: [imports](https://github.com/bbc/simorgh/blob/latest/src/server/Document/Renderers/LiteRenderer.tsx#L3-L5)
[implementation](https://github.com/bbc/simorgh/blob/latest/src/server/Document/Renderers/LiteRenderer.tsx#L14-L16) - should we include these links instead? 

Or should we include the links to the tests for the scripts (e.g. 
- https://github.com/bbc/simorgh/blob/latest/src/server/utilities/liteATITracking/index.test.ts
- https://github.com/bbc/simorgh/blob/latest/src/server/utilities/liteATITracking/viewTracking/index.test.ts
- https://github.com/bbc/simorgh/blob/latest/src/server/utilities/liteATITracking/clickTracking/index.test.ts

Reply:
If folks agree with my approach re: the PR above, I'd leave it as is, let's discuss on this PR 👍